The-Chonky-Panda/Atelier
# Reviews list skeleton UI

## Description

A review list component should exist in the ratings & review section to display all reviews that have been submitted. 
In terms of file organization, I tried to make it consistent as product overview section.
In terms of CSS, I'm using [CSS module style](https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/), since we need to have a light mode and a dark mode eventually, and import style as modules may be useful. 

## How to test

Currently the only test we have doesn't work anymore since we need to somehow mock the `fetch` method. I'll open an investigation ticket for adding tests.
For manual testing, pull this branch and start the application, the review list component should look like this:
<img width="754" alt="Screen Shot 2021-09-12 at 5 41 25 PM" src="https://user-images.githubusercontent.com/43324065/133005429-d8a460cd-5a64-4d81-b408-a5ad934455d1.png">


## Notes

Sorry for including three tickets in this PR!! Definitely not the best practice. >_< (but it makes more sense to me to bundle the review list container and individual review component together, so maybe I need to write better tickets lol)

I noticed that eslint is not giving me errors even though I'm pretty sure there are several linting problems. But I don't want to add more things to this PR so I'll open a ticket for this as well.

## Issue tracking

https://trello.com/c/kH1GdkNe
https://trello.com/c/wOLhw3Yn
https://trello.com/c/MKRfrqB0
